From 52c7bf79d447d9d015d676239cf1f2f9b2e5860c Mon Sep 17 00:00:00 2001
From: Alexander Aring <aahringo@redhat.com>
Date: Wed, 4 Aug 2021 10:15:37 -0400
Subject: [PATCH 04/14] plock: move clear waiter to debug info

This patch moves the clear waiter log message for a killed/interrupted
posix lock of a user space from error to debug. It can be confused to
see it as error in the log but it's necessary to cleanup all waiters who
are waiting for the lock.

Reported-by: Shane Bradley <sbradley@redhat.com>
---
 dlm_controld/plock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlm_controld/plock.c b/dlm_controld/plock.c
index 992fb16fef1f..692787e2b758 100644
--- a/dlm_controld/plock.c
+++ b/dlm_controld/plock.c
@@ -660,7 +660,7 @@ static void clear_waiters(struct lockspace *ls, struct resource *r,
 
 		list_del(&w->list);
 
-		log_elock(ls, "clear waiter %llx %llx-%llx %d/%u/%llx",
+		log_dlock(ls, "clear waiter %llx %llx-%llx %d/%u/%llx",
 			  (unsigned long long)in->number,
 			  (unsigned long long)in->start,
 			  (unsigned long long)in->end,
-- 
2.33.0

